### ISSUE TITLE
deal with overfitting

### ISSUE THREAD

**Issue by OliverZim** [Rob2U/adni_transformer#12]
use and experiment with the following techniques;

- data augmentation
- weight decay
- early stopping and improved checkpointing
